### DISCUSSION TITLE
Colors of branches

### DISCUSSION THREAD

**Discussion by senex77** [gitextensions/gitextensions#8432] [General] [answered]
Hi,
why are the colors of branches in git extensions reverse from what they are in git console? Remote branches are green and local branches are red, when in console (and original git bash) it is the other way around? I remember the colors could be changed in preferences and I did that, but after an update some time ago they went back to default. In documentation there is a passage about this:

"Multicolor branches
Displays branch commits in different colors if checked. If unchecked, all branches are shown in the same color. This color can be selected."

But I cannot find where to change this. Was this feature disabled? 

Regards

**Comment by pmiossec** [comment, marked as answer]
>why are the colors of branches in git extensions reverse from what they are in git console?

I share this point of view that is a little strange...

>But I cannot find where to change this. Was this feature disabled?

No, that is now handled in themes files in the "Themes" subfolder of the application.

I think you will have to create your own theme if you want to do it again.
I think that the way to go to do it is (@NikolayXHD will correct me if I am wrong):

* copy the "win10default.css" and rename the copied file to give you the name you want.
* edit the file to switch the values of `.Branch` and `.RemoteBranch`
* Restart GitExtension to select your theme in the settings

But things are moving on this feature. See #8424

We need to improve the documentation but that will wait the merger of #8424 ...

**Reply by senex77** [reply]
That did the trick. Thanks!

**Reply by NikolayXHD** [reply]
Thank you @pmiossec.
@senex77 beware that the file inside `GitExtensions\Theme` directory will be lost when updating GitExtensions to a new version, 
To prevent that, move the custom theme to `%AppData%\GitExtensions\GitExtensions\Themes\` directory.
